source/errors.ts

```diff
@@ -19,6 +19,20 @@ export class GotError extends Error {
 		Object.defineProperty(this, 'options', {
 			value: options
 		});
+
+		// Recover the original stacktrace
+		if (!is.undefined(error.stack)) {
+			const indexOfMessage = this.stack.indexOf(this.message) + this.message.length;
+			const thisStackTrace = this.stack.slice(indexOfMessage).split('\n').reverse();
+			const errorStackTrace = error.stack.slice(error.stack.indexOf(error.message) + error.message.length).split('\n').reverse();
+
+			// Remove duplicated traces
+			while (errorStackTrace.length !== 0 && errorStackTrace[0] === thisStackTrace[0]) {
```

Comment:
This assumes these stacktraces have **only** common beginning.

Reply:
Should these stack traces be separated by a line break?

Reply:
> Should these stack traces be separated by a line break to indicate that they are from two different sources?

No, that would potentially break stack trace parsers.